websites/J/JKAnime/iframe.ts

```diff
@@ -0,0 +1,16 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+
+	if(document.querySelector("video[id$='jkvideo_html5_api']") != null || document.querySelector("div.jw-media.jw-reset > video") != null) {
+		const video: HTMLVideoElement = document.querySelector("video[id$='jkvideo_html5_api']") != null
+		? document.querySelector("video[id$='jkvideo_html5_api']") : document.querySelector("div.jw-media.jw-reset > video");
+		if(video != null && !isNaN(video.duration)) {
+			iframe.send({
+				duration: video.duration,
+				currentTime: video.currentTime,
+				paused: video.paused
+			});

```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113512)